src/main/java/slimeknights/tconstruct/tools/data/Materials.java

```diff
@@ -15,63 +15,61 @@
-  public static final IMaterial slime = mat(MaterialIds.slime, true, "82c873");
-  public static final IMaterial blueslime = mat(MaterialIds.blueslime, true, "74c8c7");
-  public static final IMaterial magmaslime = mat(MaterialIds.magmaslime, true, "ff960d");
-
-  // Metals
+  //Tier 2
+  public static final IMaterial copper = mat(MaterialIds.copper, TinkerFluids.moltenCopper, true, "ed9f07");
+  public static final IMaterial searedstone = mat(MaterialIds.searedstone, TinkerFluids.searedStone, false, "ed9f07");
   public static final IMaterial iron = mat(MaterialIds.iron, TinkerFluids.moltenIron, false, "cacaca");
-  public static final IMaterial copper = mat(MaterialIds.copper, TinkerFluids.moltenCopper, false, "ed9f07");
-  public static final IMaterial pigiron = mat(MaterialIds.pigiron, TinkerFluids.moltenPigIron, false, "ef9e9b");
+  public static final IMaterial slimewood = mat(MaterialIds.slimewood, true, "82c873"); //Compsite: Green Slime -> Wood
+  public static final IMaterial slimestone = mat(MaterialIds.slimestone, true, "82c873"); //Compsite: Green Slime -> Stone
 
-  // Nether Materials
-  public static final IMaterial netherrack = mat(MaterialIds.netherrack, true, "b84f4f");
-  public static final IMaterial ardite = mat(MaterialIds.ardite, TinkerFluids.moltenArdite, false, "d14210");
-  public static final IMaterial cobalt = mat(MaterialIds.cobalt, TinkerFluids.moltenCobalt, false, "2882d4");
-  public static final IMaterial manyullyn = mat(MaterialIds.manyullyn, TinkerFluids.moltenManyullyn, false, "a15cf8");
+  //Tier 2 alternate
+  public static final IMaterial mushwood = mat(MaterialIds.mushwood, true, "24bad5");
+  public static final IMaterial blackstone = mat(MaterialIds.blackstone, true, "333333");
+  public static final IMaterial basalt = mat(MaterialIds.basalt, true, "cfcfcf");
+  public static final IMaterial bloodwood = mat(MaterialIds.bloodwood, true, "d53024");
+  public static final IMaterial witherbone = mat(MaterialIds.witherbone, true, "16202e");
 
-  // mod integration
-  public static final IMaterial bronze = mat(MaterialIds.bronze, false, "e3bd68");
-  public static final IMaterial lead = mat(MaterialIds.lead, false, "4d4968");
-  public static final IMaterial silver = mat(MaterialIds.silver, false, "d1ecf6");
-  public static final IMaterial electrum = mat(MaterialIds.electrum, false, "e8db49");
-  public static final IMaterial steel = mat(MaterialIds.steel, false, "a7a7a7");
+  //Tier 3
+  public static final IMaterial slimesteel = mat(MaterialIds.slimesteel, TinkerFluids.moltenSlimeSteel,false, "74c8c7");
+  public static final IMaterial nahuatl = mat(MaterialIds.nahuatl, false, "601cc4"); //Compsite: Obsidian -> Wood
+  public static final IMaterial ravagersteel = mat(MaterialIds.ravagersteel, TinkerFluids.moltenRavagerSteel, false, "ff5a89");
+  public static final IMaterial bronze = mat(MaterialIds.bronze, TinkerFluids.moltenBronze, false, "e3bd68");
```

Comment:
Color is too close to regular bronze. Can we make this more vibrant? like the orange slime to make it stand out more? Or maybe take the cobalt to make it purply?